tests/plugins/richcombo/manual/select.html

```diff
@@ -0,0 +1,53 @@
+<h2>Classic editor</h2>
+<div id="classic">
+	<p>I'm CKEditor 4 instance.</p>
+</div>
+<button onClick="setComboValue( 'one' )" >one</button>
+<button onClick="setComboValue( 'two' )" >two</button>
+<button onClick="setComboValue( 'three' )" >three</button>
+
+<script>
+	var combo;
+	CKEDITOR.replace( 'classic', {
+		toolbar: [ [ 'custom_richcombo' ] ],
+		on: {
+			'pluginsLoaded': function( evt ) {
+				var editor = evt.editor,
+					options = {
+						'one': 'one',
+						'two': 'two',
+						'three': 'three'
+					}
+
+				editor.ui.addRichCombo( 'custom_richcombo', {
+					panel: {
+						css: [],
+						multiSelect: false
+					},
+					options: options,
+					init: function() {
+						for ( var key in options ) {
+							this.add( key, options[ key ] );
+						}
+					},
+					onClick: function() {},
+					onRender: function() {}
+				} );
+			},
+			'instanceReady': function( evt ) {
+				var editor = evt.editor;
+
+				combo = editor.ui.get( 'custom_richcombo' );
+
+				combo.createPanel( editor );
+			}
+		}
+	} );
+
+	function setComboValue( value ) {
+		combo.select( function( option ) {
```

Comment:
This variable is defined pretty deep in code execution (`instanceReady`) which is a bit hard to find. Maybe you could use  `editor.ui.get` here and expose `editor` instance (which could be also used in `instanceReady`) when replacing the element with an editor (`CKEDITOR.replace` returns editor instance also)?